src/editables/__init__.py

```diff
@@ -1,36 +1,12 @@
-from pathlib import Path
-TEMPLATE = """\
+import inspect
 import os
-import sys
```

Comment:
Moved this to its own file so we don't have to edit this file inside a docstring